client/lib/post-normalizer/rule-content-make-images-safe.js

```diff
@@ -60,7 +60,7 @@ function isCandidateForContentImage( imageUrl ) {
 	} );
 }
 
-export default function( maxWidth ) {
```

Comment:
@samouri sure. we're resizing images passed through photon, forcing them to conform to a max width. But when we do that, we also strip a bunch of parameters also related to sizing, like crop and fit. Dropping those is what caused the reported squishing of images. 

My first thought was to modify those parameters too, instead of dropping them, but going down that road showed that altering the width of the content images actually has some pretty bad effects, especially when an image is missing a width and height in the source (which is pretty common for dotcom images and responsive wp sites). 

So the best solution I found was to leave the images alone, but to at least make them safe.